crates/services/p2p/src/config.rs

```diff
@@ -272,10 +275,11 @@ pub(crate) fn build_transport(
     };
 
     let multiplex_config = {
-        let mut mplex_config = mplex::MplexConfig::new();
-        mplex_config.set_max_buffer_size(MAX_NUM_OF_FRAMES_BUFFERED);
+        let mut mplex_config = mplex::MplexConfig::default();
+        mplex_config.set_max_buffer_size(usize::MAX);
```

Comment:
Is this really meant to be usize::MAX?

Reply:
We followed the settings parity uses here in production. In a separate PR we may consider just removing mplex altogether since it's not really used.

Reply:
Will this lead to an unbounded buffer though (well bounded by the system resources)?

Reply:
Since it still works without this setting I'll just remove it.